sdk/src/main/scala/com/horizen/certificatesubmitter/CertificateSubmitter.scala

```diff
@@ -86,10 +86,9 @@ class CertificateSubmitter
   private def checkSubmitterMessage(sidechainNodeView: View): Try[Unit] = Try {
     val signersPublicKeys = params.signersPublicKeys
 
-    val actualSysDataConstant =
-      CryptoLibProvider.sigProofThresholdCircuitFunctions.generateSysDataConstant(signersPublicKeys.map(_.bytes()).asJava, params.signersThreshold)
+    val actualSysDataConstant = params.calculatedSysDataConstant
+    val expectedSysDataConstant = getSidechainCreationTransaction(sidechainNodeView.history).getBackwardTransferPoseidonRootHash
```

Comment:
`getBackwardTransferPoseidonRootHash` name is outdated. See the comment there.

Reply:
done